### ISSUE TITLE
Validation with own generated bounding box 

### ISSUE THREAD

**Issue by joihn** [open-mmlab/mmpose#974]
the [faq](https://github.com/open-mmlab/mmpose/blob/master/docs/faq.md) says: 

- **What is `COCO_val2017_detections_AP_H_56_person.json`? Can I train pose models without it?**

  "COCO_val2017_detections_AP_H_56_person.json" contains the "detected" human bounding boxes for COCO validation set, which are generated by FasterRCNN.
  One can choose to use gt bounding boxes to evaluate models, by setting `use_gt_bbox=True` and `bbox_file=''`. Or one can use detected boxes to evaluate
  the generalizability of models, by setting `use_gt_bbox=False` and `bbox_file='COCO_val2017_detections_AP_H_56_person.json'`.
  
  

But how do validate my model with it's **own generated bounding box**, without any external bounding info (not even from the ground truth) ? 
 In other words, I want my validation metrics to contain both metric about the pose regression quality **and** the generated bounding box regression quality.
 
 Is this possible ? (for info I'm currently using a TopDown heatmap model for 2D RGB human pose)

**Comment by ly015** [comment]
As described in the FAQ, `COCO_val2017_detections_AP_H_56_person.json` is a file that contains detection results from a detection model. This file is only used in validation and test, so you can train a pose model without it.

If I understand it right, you want to generate bboxes from the keypoint predictions, and evaluate both bbox and keypoint. I would suggest implementing a new evaluation metric to do it. Please refer to examples in [core/evaluation/top_down_eval.py](https://github.com/open-mmlab/mmpose/blob/master/mmpose/core/evaluation/top_down_eval.py).

**Comment by joihn** [comment]
Hum, I think I have a misunderstanding.
I thought that top-down model from MMpose could do both **object detection** and then **pose detection** within the object.

But apparently I'm wrong ? MMpose models can only do pose detection wihtin a tight box, and therefore always need object location (=BBox) from another net ?

**Comment by joihn** [comment]
okay, thanks for your help
